VirtoCommerce.Platform.Web/Controllers/HomeController.cs

```diff
@@ -27,7 +26,7 @@ public ActionResult Index()
             var version = PlatformVersion.CurrentVersion.ToString();
             var demoCredentials = ConfigurationHelper.GetAppSettingsValue("VirtoCommerce:DemoCredentials");
             var resetTimeStr = ConfigurationHelper.GetAppSettingsValue("VirtoCommerce:DemoResetTime");
-            var license = LoadLicense();
+            var license = LicenseProvider.LoadLicense();
```

Comment:
If we have a "License Provider" then would be enough to name this method just "Load"